### PR TITLE
Add unit tests for ssg/build_renumber.py

### PR DESCRIPTION
**Pull request by jan-cerny**
#### Description:
Add unit tests for ssg/build_renumber.py
#### Rationale:
increase testing of build scripts

### CODE REVIEW

**Comment by jan-cerny** [comment]
I have add a fix for the Code climate.

**Comment by jan-cerny** [comment]
For some unknown reason the CI is stuck. Can you please rebase and push force it?

**Comment by jan-cerny** [comment]
I have rebased this PR on the top of the latest upstream master branch.

File: tests/unit/ssg-module/test_build_renumber.py [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+
+def test_check_and_correct_xccdf_to_oval_data_export_matching_constraints_5(
+        xccdf_with_value, oval_with_external_variable):
+    ssg.build_renumber.check_and_correct_xccdf_to_oval_data_export_matching_constraints(
+        xccdf_with_value, oval_with_external_variable)
+    assert xccdf_with_value is not None
+
+
+@pytest.fixture
+def xccdf_with_invalid_cce():
```
Comment (matejak) [inline tests/unit/ssg-module/test_build_renumber.py:368]: Would it be possible to add a test of `verify_correct_form_of_referenced_cce_identifiers` with a rule without a CCE?

**Review by matejak — CHANGES_REQUESTED** [review]
Looks great, just one small request.

**Comment by jan-cerny** [comment]
I have add a test with a rule without CCE

**Review by matejak — APPROVED** [review]

**Comment by matejak** [comment]
Perfect, great test coverage. Thank you for the PR!

**Pull request merged**